tcti/tcti_device.c

```diff
@@ -251,8 +250,6 @@ TSS2_RC InitDeviceTcti (
     }
     else
     {
-        (*tpmLocalTpmPrintf)(NO_PREFIX, "Initializing %s Interface\n", interfaceName );
```

Comment:
Seems like you could init callback before this and use it instead of the tpmLocalTpmPrintf pointer.